components/brave_shields/browser/brave_shields_util.h

```diff
@@ -25,6 +25,19 @@ class ProfileIOData;
 
 namespace brave_shields {
 
+enum class ShieldState {kBlock, kBlock3rd, kAllow};
```

Comment:
why don't we just assign string values to these enums? Then we wouldn't need `ShieldStateFromString` and other checks against strings

Reply:
I think this code is being refactored soon so I'm not against that but not as part of this rebase.

Reply:
Edit: bbondy: removed my comment, it was not valid.

Reply:
hmm, what do you mean by assigning string values? Not sure if this is possible in C++ :-/

Reply:
@bridiver PTAL?